ajax/libs/brython/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "brython",
+  "filename": "brython.js",
+  "version": "3.3.2",
+  "description": "Python 3 in the browser",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brython-dev/brython.git"
+  },
+  "keywords": [
+    "Python",
+    "browser"
+  ],
+  "author": "Pierre Quentel",
+  "license": "BSD-2-Clause",
```

Comment:
It was a typo, the licence is BSD-3. I have updated [package.json](https://github.com/brython-dev/brython/blob/master/npm/package.json) in the Brython repository